rpc/dial.go

```diff
@@ -75,7 +79,7 @@ func dialInner(
 	}
 	if cached {
 		if dOpts.debug {
-			logger.Debugw("connected (cached)", "address", address)
+			logger.Infow("connected (cached)", "address", address)
```

Comment:
```suggestion
			logger.Debugw("connected (cached)", "address", address)
```

changed for testing purposes